addons/account/models/company.py

```diff
@@ -15,6 +15,9 @@
 class ResCompany(models.Model):
     _inherit = "res.company"
 
+    def _get_invoice_reference_type(self):
```

Comment:
_get_invoice_reference_typeS() also please change the function _get_reference_type() to _get_reference_typeS()

it is sooo misleading!